app/views/dashboards/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Dashboard#index</h1>
-<p>Find me in app/views/dashboard/index.html.erb</p>
+<div class="container mt-5">
+  <h1 class="text-center mb-md-5">
+    <%= @survey.title %>
+  </h1>
+
+  <script>
+    let Evaweb = {};
```

Comment:
Don‘ use ecmascript 6 in inline scripts as some browsers don‘t fully support it yet